app/backends.py

```diff
@@ -19,32 +19,34 @@ def configure_user(self, user: User) -> User:
 
         :returns: [User] The user object with updates to is_staff, email, first_name, last_name fields.
         """
-        url = "https://yalies.io/api/people"
-        querystring = {"query": user.username}
+        url = 'https://yalies.io/api/people'
+        payload = {
+            'filters': {
+                'netid': user.username,
+            }
+        }
         headers = {
-            'Authorization': f"Bearer {get_secret('YALIES_API')}",
+            'Authorization': 'Bearer ' + get_secret('YALIES_API'),
         }
 
         try:
-            response = requests.post(url, headers=headers, json=querystring)
+            response = requests.post(url, headers=headers, json=payload)
             response.raise_for_status()
             data = response.json()[0]
-            
+
             email = data['email']
             first_name = data['first_name']
             last_name = data['last_name']
             org = data['organization']
-            is_staff = False
-            if org is not None:
-                is_staff = True
+            is_staff = (org is not None)
```

Comment:
More compact. You could also consider merging this block into the one below rather than declaring intermediary variables.

Reply:
Update: I did that